include/aws/common/config.h.in

```diff
@@ -29,5 +29,6 @@
 #cmakedefine AWS_ARCH_INTEL
 #cmakedefine AWS_ARCH_INTEL_X64
 #cmakedefine AWS_USE_CPU_EXTENSIONS
+#cmakedefine AWS_HAS_POSIX_BOOT_CLOCK
```

Comment:
This header is part of the "public" API

If we don't need to know this anywhere except in common/source/posix/clock.c, consider making it "private" via `target_compile_definitions()` that only apply to aws-c-common

Reply:
Removed